packages/wallet/src/redux/protocols/indirect-funding/player-a/__tests__/stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Modal from 'react-modal';
+import { Provider } from 'react-redux';
+import IndirectFundingContainer from '../../../../../containers/indirect-funding/indirect-funding';
```

Comment:
Would be nice to move the container into the `protocols/indirect-funding` folder.

Reply:
Addressed in https://github.com/magmo/apps/pull/379.